modules/imaging_browser/php/queryengine.class.inc

```diff
@@ -75,6 +75,15 @@ class QueryEngine extends \LORIS\Data\Query\SQLQueryEngine
                 new Cardinality(Cardinality::MANY),
                 $ScanType,
             );
+            $items[] = new QCDictionaryItem(
+                $ScanType . "_Selected" ,
+                "Selected file for $ScanType for session",
+                $scope,
+                new \LORIS\Data\Types\BooleanType(),
+                new Cardinality(Cardinality::MANY),
```

Comment:
Note from meeting:
Cardinality is in the context of the scope above (in this case session)